packages/compass-crud/src/components/breadcrumb.jsx

```diff
@@ -32,23 +32,23 @@ class BreadcrumbComponent extends React.PureComponent {
   render() {
     return (
       <div className={`${BEM_BASE}-container`}>
-        <div onClick={this.onHomeClicked.bind(this)} className={`${BEM_BASE}-tab`}>
+        <button onClick={this.onHomeClicked.bind(this)} className={`${BEM_BASE}-tab`}>
           <FontAwesome name="home" className={`${BEM_BASE}-home-icon`}/>
           {this.props.collection}
-        </div>
+        </button>
         {this.props.path.map((name, i) => {
           let displayName = '';
           if (typeof name === 'number' && i > 0) {
             displayName = this.props.path[i - 1] + '.';
           }
           displayName = displayName.concat(name);
           return (
-            <span
+            <button
```

Comment:
Fixes eslint `jsx-a11y/no-noninteractive-element-interactions`
<img width="341" alt="Screen Shot 2022-05-19 at 11 46 42 PM" src="https://user-images.githubusercontent.com/1791149/169448710-68c69b54-4796-40ca-94a2-9ce24662173a.png">
Same with the `button` update above.